### ISSUE TITLE
Problems fitting screen size

### ISSUE THREAD

**Issue by PhilippSua** [tomahawk-player/tomahawk#396]
In some cases the player switches from normal screen fitting size to some other layout where the bottom playbar is not completely on the screen anymore and it's not possible to go back to fullscreen layout.
This happens for example on the page of an album when you click on the currently playing track in this albums tracklist (the one showing the red progressbar). In this case it switches back to the normal layout when cklicking on another entry in the tracklist.

**Comment by albertocalmeida** [comment]
It happens to me randomly (or so it seems) that Tomahawk starts in similar fashion: the weird "fullscreen" version has a titlebar and occupies the full width of the active workspace but goes beyond the height of the screen so it is seen on the workspace below. Though resizeable this height can't be reduced. 

Due to file caching, restarting Tomahawk isn't enough to run a clean restart. I've recently found that clearing cache does the trick, but being a total noob I refrain from doing recommendations.

I'm running Ubuntu 15.10.

**Comment by theli-ua** [comment]
yeah, explodes exither vertically or horizontally for me all the time

**Comment by muesli** [comment]
If you could find a way to reproduce that behaviour, that would help me a lot.

**Comment by KorribanMaster** [comment]
I'm having the same issue on Linux Mint 17.3 Cinnamon with Tomahawk 0.8.4. Appears every time after closing tomahawk window and reopening it while still playing music. If you need further information ask as this problem is rather annoying.

**Comment by anirudha-banerjee** [comment]
Facing same issue on elementary os freya with tomahawk 0.8.4. Happens every time i open it.

**Comment by tnga** [comment]
same behavior on ubuntu 14.04. The fact is, when tomahawk is opened and you navigate through sidebar panel (collections, ...), the window automatically resize itself (maybe to fit content) and it can not be resized (even maximized) anymore, unless you restart the player. This is frustrating when your screen  size is smaller than 1280px.

**Comment by TunableRoom9** [comment]
Same problem here with Elementary OS

**Comment by pklausner** [comment]
Same probleme here on Ubuntu 16.04. On start-up, the height fits the screen.  After a split second it starts rendering content in the main viewpane, then increases the height and pushes the controls off screen.  Only flushing .local/share/Tomahawk did restore function.
